aws/cloudformation/components/database.yml.erb

```diff
@@ -148,14 +157,48 @@
       Parameters:
         # BEGIN: Static configuration settings. Changing these settings will trigger a restart of each database instance.
         performance_schema: 1
+        # END: Static configuration settings.
+
+        # BEGIN: Dynamic configuration settings. Changing these settings does NOT require nor trigger restart of instances.
+
+        # Prevent a slow query from degrading instance or cluster performance.
+        # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_execution_time
+        max_execution_time: 30000
```

Comment:
It looks like we are removing the last place referencing this parameter group, is there a reason we are updating the max_execution_time on it?

Reply:
Suresh noted in the description, but existing instances will continue to use the parameter group they were created with. These changes here will modify the parameter group being used.

This PR is doing three things things.
1. editing a some settings in the in-use Aurora 2 parameter group
2. creating a new Aurora 3 parameter group
3. removing the per-instance specification of a parameter group (a no-op on existing instances)

A follow up PR will
4. specify the default parameter group for new instances via the cluster

@sureshc This PR description talks a lot about item 3, but all the risk and service interruption is coming from item 1, right?